charts/federator/templates/configmap.yaml

```diff
@@ -61,3 +66,11 @@ data:
         {{- end}}
     {{- end }}
     {{- end }}
+
+  # TODO: add validation and fail early during templating: either contents should be provided; or explicitly system trust store enabled
+  {{- if .Values.remoteCAContents }}
+  remote-ca.pem: {{ .Values.remoteCAContents | quote }}
```

Comment:
I'm not sure this will work: This produces a quoted string containing `\n` characters. Not sure this will lead to actual new lines on disk. So parsing of these certificates might be broken. 

You can test the yaml output using 

```
FEDERATION_DOMAIN_BASE=example.com ./hack/bin/selfsigned-kubernetes.sh
make helm-template-wire-server
```

Reply:
The yaml of the expanded configmap contains a quoted string with escaped newlines, as you say, but on the volume using this configmap this translates into a file where those escaped newlines have turned into actual newlines. I've checked it by attaching to a running federator pod.